skbio/tree/_tree.py

```diff
@@ -617,6 +617,72 @@ def lowest_common_ancestor(self, tipnames):
 
     lca = lowest_common_ancestor  # for convenience
 
+    def path(self, other, include_ends=False):
+        r"""Return the list of node names in the path from one node to another.
```

Comment:
There is an exception to handle: What if there is no path connecting self and other? This could happen if they are not in the same tree.